pkg/controller/job/job_controller_test.go

```diff
@@ -1859,10 +1865,10 @@ func hasTrueCondition(job *batch.Job) *batch.JobConditionType {
 
 func TestSyncPastDeadlineJobFinished(t *testing.T) {
 	clientset := fake.NewSimpleClientset()
-	manager, sharedInformerFactory := newControllerFromClient(clientset, controller.NoResyncPeriodFunc)
+	fakeClock := &clocktesting.FakeClock{}
```

Comment:
nit:
```suggestion
	fakeClock := clocktesting.NewFakeClock(time.Now())
```

Otherwise you might be starting with a unix timestamp of 0. Then I'm not sure what is happening below when you say `Unix()-1`

Reply:
/Ack. Changed accordingly.